cmd/ship_receiver_plugin.cpp

```diff
@@ -52,7 +52,10 @@ class ship_receiver_plugin_impl : std::enable_shared_from_this<ship_receiver_plu
             SILK_CRIT << "SHiP initial read failed : " << ec.message();
             sys::error();
          }
-         abi = load_abi(eosio::json_token_stream{(char*)buff.data().data()});
+         auto end = buff.prepare(1);
+         ((char *)end.data())[0] = '\0';
+         buff.commit(1);
```

Comment:
This seems rather odd that it would not always contain the terminating character.

Reply:
Yeah, I am not sure whether we should fix it like this or not. 

I don't think boost websocket scream will write a zero there. So maybe we should fix json_token_stream. 

But I am not sure if it's a good idea to touch cdt for this issue..